src/motorControl.cpp

```diff
@@ -199,9 +224,17 @@ void MotorControl::exec() {
             platform->startCooldown();
         }
     }
+    if (motorTest) {
+        // testing motor ramp
+        Serial.println("Motor testing......");
+        Serial.print("Current Speed: ");
+        Serial.println(motor->getCurrentSpeed());
+        if(motor->isTargetVelocityReached())
+            stopMotion();
+    }
 
     // If past end time...
-    if (millis() >= state.run_end) {
+    if (!motorTest && millis() >= state.run_end) {

```

Comment:
Is this check for motorTest necessary?

Reply:
Without it, the motor gets stopped before it's started